pontoon/administration/static/js/admin_project.js

```diff
@@ -194,7 +194,7 @@ $(function () {
 
     try {
       projectLocales = JSON.parse($(this).val());
-    } catch (error) {
+    } catch {
```

Comment:
Fixing this via eslint comment turned out to be a nightmare

```js
        try {
          data = JSON.parse(event.data);
        }
        // eslint-disable-next-line no-unused-vars
        catch (_) {
          return;
        }
```

This works, but prettier will move the comment with the catch block, breaking eslint.

Started looking into https://github.com/prettier/eslint-plugin-prettier, but that's a rabbit hole I don't want to go down for these 2 errors.

Reply:
As we're not using the variables, eliding the `(error)` or `(_)` completely is the right choice.